### DISCUSSION TITLE
How to set Hourly Duration ?

### DISCUSSION THREAD

**Discussion by Hisashi-Reyuga** [pranjal-joshi/Screeni-py#137] [Q&A] [answered]
I would like to screen stocks on hourly candle duration & I tried numbers like 0.1 and  0.04166666666 and even 0.16260162601 for duration parameter but it doesn't seem to provide results if the parameter is less than 1 so could you help me with this one?

**Comment by pranjal-joshi** [comment, marked as answer]
Hi @Hisashi-Reyuga 

Please edit parameters in the `screenipy.ini` file as per the following example:
```
duration = 1d        // For Daily candle
duration = 1h        // For hourly candle
duration = 15m      // 15m Intraday candle
duration = 1w       // Weekly candle
```

Follow the Candle timeframe formats just like TradingView.

**Reply by Hisashi-Reyuga** [reply]
Thank you so much
